packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Button as MuiButton } from "@mui/material";
+import type { ButtonProps as MuiButtonProps } from "@mui/material";
+import { Tooltip } from "./";
+import { memo } from "react";
+
+export type ButtonProps = {
+  endIcon?: React.ReactNode;
+  id?: string;
+  isDisabled?: boolean;
+  isFullWidth?: boolean;
+  onClick?: MuiButtonProps["onClick"];
+  size?: MuiButtonProps["size"];
+  startIcon?: React.ReactNode;
+  text?: string;
+  /**
+   * tooltipText determines the text of the tooltip
+   * that wraps the button if it's icon-only
```

Comment:
These can be in one line. In VSCode, I'm pretty sure these will already have line-breaks in the docs, and then another line-break from the one you added.

```suggestion
   * `tooltipText` determines the text of the tooltip that wraps the button if it's icon-only.
```